src/state/actions/index.js

```diff
@@ -19,44 +19,44 @@ export const ADD_ITEM_IMAGE_START = 'ADD_ITEM_IMAGE_START';
 export const ADD_ITEM_IMAGE_SUCCESS = 'ADD_ITEM_IMAGE_SUCCESS';
 export const ADD_ITEM_IMAGE_ERROR = 'ADD_ITEM_IMAGE_ERROR';
 
-export const fetchProducts = authState => dispatch => {
+export const fetchProducts = (authState) => (dispatch) => {
   let oktaStore = JSON.parse(localStorage['okta-token-storage']);
   let oktaId = oktaStore.idToken.claims.sub;
   dispatch({ type: FETCH_PRODUCTS_START });
   getDSData(
     `${process.env.REACT_APP_API_URI}items/profile/${oktaId}`,
     authState
   )
-    .then(response => {
+    .then((response) => {
```

Comment:
Did we discuss this as a team? I'm open to this format but lets talk.

Reply:
am not completely sure but i think that's the prettier addon on adam's VSCode, maybe I'm wrong but think that's the problem with the style change there

Reply:
This has been fixed. I figured out that my prettierrc file was overriding the one in the repo, and changed a setting to avoid that going forward.